.github/codecov.yml

```diff
@@ -9,7 +9,7 @@ coverage:
     patch:
       default:
         target: 80%
-        informational: true
+        informational: false
```

Comment:
Probably we need to do this for spanner as well

Reply:
Added